src/main/java/dinkplugin/domain/PlayerLookupService.java

```diff
@@ -0,0 +1,36 @@
+package dinkplugin.domain;
+
+public enum PlayerLookupService {
+    NONE("None"),
+    OSRS_HISCORE("OSRS HiScore"),
+    CRYSTAL_MATH_LABS("Crystal Math Labs"),
+    TEMPLEOSRS("Temple OSRS"),
+    WISEOLDMAN("Wise Old Man");
```

Comment:
Also, could you please rename these enums to `TEMPLE_OSRS` and `WISE_OLD_MAN`, respectively?